Framework/python/ldmxcfg.py

```diff
@@ -925,4 +925,15 @@ def __str__(self):
 
         return msg
 
-    
+class RunHeaderAna(Analyzer) :
+    """                                                                                                                  
+    Contains an instance of RunHeaderAnalyzer that
+    has already been configured.
+
+    Examples
+    --------
+        p.sequence.append( dqm.RunHeaderAna() )
+    """
+
+    def __init__(self, name='RunHeaderAnalyzer'):
+        super().__init__(name, 'RunHeaderAnalyzer', '')  
```

Comment:
The last argument is the module that the processor is in. This gets used to deduce the library path that should be loaded.

```suggestion
        super().__init__(name, 'RunHeaderAnalyzer', 'Framework')
```